### PR TITLE
Group revisions by created_at and user

### PR DESCRIPTION
**Pull request by rubenvanerk**
Grouping all changes by user and timestamp seems more logical. 

We talked about this on Slack about a month ago. Let me know what you think.

### CODE REVIEW

**Comment by Samuell1** [comment]
Hey, thanks! I will review it soon.

File: formwidgets/RevisionHistory.php [hunk 1]
```diff
@@ -51,8 +56,12 @@ public function onRevertHistory()
         $section = $modelClass::find($this->model->id);
 
         $revision = Revision::find(input('revision_id'));
+        $revisions = Revision::where('user_id', $revision->user_id)->where('created_at', $revision->created_at)->get();
```
Comment (Samuell1) [inline formwidgets/RevisionHistory.php]: Why was this changed? Because now it reverts all changes by that user not a specific that user selected.

Comment (rubenvanerk) [reply formwidgets/RevisionHistory.php]: The idea is that a user selects a group of changes that was made at the same time. These changes will then all rollback at the same time.

Comment (Samuell1) [reply formwidgets/RevisionHistory.php]: Then it should be made as separate option for multiple and single one.

Comment (rubenvanerk) [reply formwidgets/RevisionHistory.php]: I added checkboxes before each revision to select the ones you want to restore.
